doc/7/protocols/websocket/introduction/index.md

```diff
@@ -13,3 +13,11 @@ Inherits from: [KuzzleEventEmitter](/sdk/js/7/core-classes/kuzzle-event-emitter)
 The WebSocket protocol can be used by an instance of the SDK to communicate with your Kuzzle server.
 
 This protocol allows you to use all the features of Kuzzle, including [real-time notifications](/sdk/js/7/essentials/realtime-notifications).
+
+<SinceBadge version="2.10.0"/>
+:::info
```

Comment:
```suggestion
::: info
```

Reply:
I wish we could have a dedicated section so we can send links directly to people asking about this feature